app/src/main/kotlin/com/backbase/golden_sample_app/ui/theme/Color.kt

```diff
@@ -0,0 +1,11 @@
+package com.backbase.golden_sample_app.ui.theme
```

Comment:
I would recommend to delete the unused functionality or code since we want to minimize the code that is actually needed to set up the golden sample app,
in case this is needed in the future add it in the next PR